### PR TITLE
mdcat: update to 0.23.0

### PR DESCRIPTION
**Pull request by FollieHiyuki**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/mdcat/template [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+post_install() {
+	bash_completion=$(find ${wrksrc}/target -name mdcat.bash -print -quit)
+	vcompletion ${bash_completion} bash
+	zsh_completion=$(find ${wrksrc}/target -name _mdcat -print -quit)
+	vcompletion ${zsh_completion} zsh
+	fish_completion=$(find ${wrksrc}/target -name mdcat.fish -print -quit)
+	vcompletion ${fish_completion} fish
+
+	man_page=$(find ${wrksrc}/target -name mdcat.1 -print -quit)
+	vman ${man_page}
```
Comment (ericonr) [inline srcpkgs/mdcat/template:25]: Dirty deeds done dirt cheap :p

**Review by ericonr — COMMENTED** [review]
If you're going to merge, you need to add a broken block, grep `ftbfs in ring`.

File: srcpkgs/mdcat/template (hunk 1 above)
Comment (FollieHiyuki) [reply srcpkgs/mdcat/template:30]: It's cargo's fault to dump things deep inside /target/release/build xD

**Comment by FollieHiyuki** [comment]
Added a broken block as you suggested.

**Pull request closed without merging**